lib/ansiblelint/skip_utils.py

```diff
@@ -0,0 +1,164 @@
+# (c) 2019, Ansible by Red Hat
+#
+# Permission is hereby granted, free of charge, to any person obtaining a copy
+# of this software and associated documentation files (the "Software"), to deal
+# in the Software without restriction, including without limitation the rights
+# to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+# copies of the Software, and to permit persons to whom the Software is
+# furnished to do so, subject to the following conditions:
+#
+# The above copyright notice and this permission notice shall be included in
+# all copies or substantial portions of the Software.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+# AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+# LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+# OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+# THE SOFTWARE.
+
+"""Utils related to inline skipping of rules."""
+from itertools import product
+import logging
+
+import ruamel.yaml
+
+INLINE_SKIP_FLAG = '# noqa '
+
+_logger = logging.getLogger(__name__)
+
+
+def get_rule_skips_from_line(line):
+    """Return list of rule ids skipped via comment on the line of yaml."""
+    rule_id_list = []
+    if INLINE_SKIP_FLAG in line:
```

Comment:
Also could reverse this check into a guard style:
```suggestion
    if INLINE_SKIP_FLAG not in line:
        return []
```

Reply:
I would like to apply changes from [comment here](https://github.com/ansible/ansible-lint/pull/529#discussion_r291989753) as you suggested above and preserve func current behavior.